knowledge-base/json-length-is-too-large-net-core.md

```diff
@@ -0,0 +1,80 @@
+---
+title: System.ArgumentException The JSON value of length n is too large and not supported
+description: The error message System.ArgumentException The JSON value of length n is too large and not supported is displayed in the report viewer 
+type: troubleshooting
+page_title: System.ArgumentException The JSON value of length n is too large and not supported
+slug: json-length-is-too-large-net-core
+position: 
+tags: 
+ticketid: 1556924
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+		<tr>
+			<td>Project Type</td>
+			<td>ASP.NET Core</td>
+		</tr>
+		<tr>
+			<td>Report Viewers</td>
+			<td>HTML5-based Report Viewers</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+Rendering a report with a lot of data, resulting a big number of pages, would result in displaying the "System.ArgumentException: The JSON value of length n is too large and not supported"
```

Comment:
To avoid repeating 'resulting' may change, for example, to ', hence a big number of pages, ' or similar